apps/api-documenter/src/utils/Utilities.ts

```diff
@@ -7,6 +7,7 @@ import {
 } from '@microsoft/api-extractor-model';
 
 export class Utilities {
+  private static readonly _badFilenameCharsRegExp: RegExp = /[^\w\-\.]/ig;
```

Comment:
Can you include a comment that mentions what this matches. Something like:

```suggestion
  /**
   * This RegExp matches any any character that is not:
   * * Alphanumeric: (a-z, A-Z, 0-9)
   * * A hyphen (-)
   * * A period (.)
   * * An underscore (_)
   */
  private static readonly _badFilenameCharsRegExp: RegExp = /[^\w\-\.]/ig;
```

Reply:
I expanded `\w` to be `a-z0-9_` which captures the same idea, and is more obvious.

The world of regular expressions is http://regex101.com
 1. You can't write general regular expressions without using a bunch of weird operators
 2. Nobody can ever remember what the operators do

Thus,  anyone who works on RegExp code does it using a visualizer tool.